Podfile.lock

```diff
@@ -21,28 +21,19 @@ PODS:
 
 DEPENDENCIES:
   - AeroGear-Push-Swift (= 1.2.0)
-  - AeroGearHttp (from `https://github.com/aerogear/aerogear-ios-http.git`, branch
-    `master`)
+  - AeroGearHttp (= 0.5.1)
   - OCMock (= 3.2)
   - OHHTTPStubs (= 4.7.1)
   - OHHTTPStubs/Swift (= 4.7.1)
   - ReachabilitySwift (= 2.3.3)
 
-EXTERNAL SOURCES:
-  AeroGearHttp:
-    :branch: master
-    :git: https://github.com/aerogear/aerogear-ios-http.git
-
-CHECKOUT OPTIONS:
-  AeroGearHttp:
-    :commit: 20e954caa168ed8605ae05322fbfaf038fd79366
-    :git: https://github.com/aerogear/aerogear-ios-http.git
-
 SPEC CHECKSUMS:
   AeroGear-Push-Swift: 2b6ada18939e58d003e6a2315c22be22556048b1
-  AeroGearHttp: f3e362a7f9e15abcd914d5c3df68999880b22f18
+  AeroGearHttp: 9eb405b694aa9ac5daff842f68648f4a9fe0fa66
   OCMock: 28def049ef47f996b515a8eeea958be7ccab2dbb
   OHHTTPStubs: f7d1604d04d37d055460c0ef9af01bb249ded0fa
   ReachabilitySwift: e0f6b6f280effb47731acfaaa2d5ffe223703793
 
-COCOAPODS: 0.39.0
+PODFILE CHECKSUM: 2555b1456228d77d901c6d8ae3425961035d4516
+
+COCOAPODS: 1.0.1
```

Comment:
@danielpassos this one need to be updated too to 0.39.0 
Make sure locally you do use 0.39.0

```
pod _0.39_ install
```